examples/commoncrawl/flows/mapping.flow.json

```diff
@@ -0,0 +1,42 @@
+{
```

Comment:
I like for these to have as little cruft as possible so they're easier to understand - can remove description, stopOnError, options, versions; and then from the step - additionalCollections, validateEntity, sourceCollection, customHook, retryLimit, batchSize, threadCount